docs/concepts/storage/volumes.md

```diff
@@ -320,6 +321,18 @@ simultaneous writers allowed.
 
 See the [iSCSI example](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/iscsi) for more details.
 
+### fc(fibre channel)
+
+A `fc` volume allows an existing fibre channel volume to be mounted into your pod.
+You can specify single or multiple target World Wide Names to the parameter
+targetWWNs in your volume configuration. If multiple WWNs are specified,
+targetWWNs expects that those WWNs form multipath connection.
+
+__Important: You must have your own FC Storage setup running with the volume
```

Comment:
FC SAN Zone manager must allocate and mask those LUNs to the target  WWNs so Kubernetes hosts can access them.